### ISSUE TITLE
OTA Update Error

### ISSUE THREAD

**Issue by lucaobermaier** [jomjol/AI-on-the-edge-device#1563]
### The Problem

Evertyime when I try to use the OTA I get the following error:

Server closed the connection abruptly!

What am I doing wrong? Or how can I solve this issue?
What are other steps for updating - maybe I did not get the update wiki right?!

Best regards

### Version

13.0.5

### Logfile

```shell
[       4] 2022-12-12T16:41:07 <INF> [MAIN] =================================================
[       4] 2022-12-12T16:41:07 <INF> [MAIN] ==================== Startup ====================
[       4] 2022-12-12T16:41:07 <INF> [MAIN] =================================================
[       4] 2022-12-12T16:41:07 <INF> [MAIN] Branch: 'HEAD', Revision: e23c89a, Date/Time: 2022-12-06 19:34, Web UI: Release: v13.0.5 (Commit: e23c89a)
[       4] 2022-12-12T16:41:07 <INF> [MAIN] Reset reason: Via esp_restart
[       8] 2022-12-12T16:41:11 <INF> [MAIN] =================================================
[       8] 2022-12-12T16:41:11 <INF> [MAIN] ================== Main Started =================
[       8] 2022-12-12T16:41:11 <INF> [MAIN] =================================================
[      10] 2022-12-12T16:41:13 <INF> [MAIN] Initialization completed successfully!
[      15] 2022-12-12T17:41:18 <ERR> [MQTT IF] Init with no Client_ID () or Last Will Topic (). Abort Init!
[      15] 2022-12-12T17:41:18 <ERR> [MQTT IF] Init failed, skipping all MQTT publishings in this round!
```

**Comment by caco3** [comment]
What version is running while you try to update?
In 12.0.1 there still is a bug in OTA. The best is to unzip firmware.zip and upload the firmware.bin directly.

also another reason could be a weak or unstable Wifi connection.

**Comment by yuri-vashchenko** [comment]
I have the same issue. After upgrading from `11.3.1` to `12.0.1` I cannot upgrade to `13.0.8`.
When I select _any_ file (including `update.zip` or `firmare.bin`) and press `Upload and upgrade` I always get the same error `Server closed the connection abruptly!`

My Wi-Fi is strong. The system does not even try to start uploading the file

**Comment by lucaobermaier** [comment]
> You can use the [Web Console](https://jomjol.github.io/AI-on-the-edge-device/index.html) to get USB log from the device. The USB log contains more information about the startup and operation of the device than the normal Web UI log

Ausgabe: firmware directory deleted - v2

W (38174) httpd_txrx: httpd_sock_err: error in recv : 104

**Comment by caco3** [comment]
Please provide the full log (from startup)

Alternatively use the Web Installer to set it up again properly.

**Comment by yuri-vashchenko** [comment]
Looks like to use the Web Installer I have to connect my device to PC, but it is installed on my water meter. Is there any way to upgrade it remotely?

**Comment by caco3** [comment]
Yes, if OTA does not work, you have to connect it to your PC using USB
